packages/core/tests/error.test.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  DiagnosticError,
+  isDiagnosticError,
+  Loggable,
+  Log,
+  LogOptions,
+  LogLevel,
+} from '..';
+
+const content: Loggable = ['This is what happened', 'and this'];
+
+describe('DiagnosticError', () => {
+  const title = 'An error happened';
+  const suggestion: Loggable = 'This is what you can do about it';
+  it('defines a DiagnosticError', () => {
+    const err = new DiagnosticError({title, content, suggestion});
+    expect(isDiagnosticError(err)).toBe(true);
+    expect(err.title).toMatch(title);
+    expect(err.content).toMatchObject(content);
+    expect(err.suggestion).toMatch(suggestion);
+    expect(err).toBeInstanceOf(Error);
+    expect(err.name).toMatch(Error.name);
+    expect(err.message).toMatch(title);
+    expect(err.stack).toBeDefined();
+  });
+});
+describe('Loggable', () => {
```

Comment:
I don't think this is actually testing any real source? I would just leave it out. 
Log tests should live in the [cli package](https://github.com/ismail-syed/sewing-kit/blob/main/packages/cli/src/common.ts#L266).

Reply:
The Log and Loggable interfaces are in core, so I put tests for them here.  This test isn't actually logging anything, the implementation of the Log function is mocked out.  I was basically just exercising the interface, and how a Log method could be defined and used.  Would it be okay to leave this here and the more e2e tests for the cli would live in the cli package and actually test logging?

Reply:
I updated the test name and the test so it makes more sense.